enterprise/server/workflowconf/test_data/basic.yaml

```diff
@@ -0,0 +1,10 @@
+actions:
+  - name: "Build and test"
+    triggers:
+      push:
+        branches: [main]
+      pull_request:
+        branches: [main]
+    bazel_commands:
+      - build //...
+      - test //...
```

Comment:
This looks awesome, really happy where we netted out!

Reply:
Same -- nice and simple!